content/en/apps/guides/hosting/4.x/app-developer.md

```diff
@@ -0,0 +1,144 @@
+---
+title: "App Developer Hosting in CHT 4.x"
+linkTitle: "App Developer Hosting"
+weight: 40
+aliases:
+- /apps/guides/hosting/app-developer
+description: >
+  Hosting the CHT when developing apps
+---
+
+{{% alert title="Note" %}} This guide assumes you are a CHT app developer wanting to either run concurrent instances of the CHT, or easily be able to switch between different instances without loosing any data while doing so. To do development on the CHT core itself, see the [development guide]({{< relref "apps/guides/hosting/core-developer.md" >}}).
+
+To deploy the CHT 3.x in production, see either [AWS hosting]({{< relref "apps/guides/hosting/3.x/self-hosting.md" >}}) or [Self hosting]({{< relref "apps/guides/hosting/3.x/ec2-setup-guide.md" >}}). 4.x production hosting guides are coming soon!{{% /alert %}}
+
+
+## Getting started
+
+Be sure to meet the [CHT hosting requirements]({{< relref "apps/guides/hosting/requirements" >}}) first. As well, if any other CHT 4.x instances using Docker Compose are running locally, stop them otherwise port, storage volume and container name conflicts may occur. To stop ALL containers, you can use `docker kill $(docker ps -q)`.
```

Comment:
```suggestion
Be sure to meet the [CHT hosting requirements]({{< relref "apps/guides/hosting/requirements" >}}) first. To avoid conflicts, ensure that all other CHT 4.x instances are stopped. To stop ALL containers, you can use

```shell
docker kill $(docker ps -q)
```

```

Reply:
That suggestion didn't get formatted correctly. Suggest new line for the code so it doesn't get affected by a line break.